trendmicro_apex/icon_trendmicro_apex/actions/add_file_to_udso_list/action.py

```diff
@@ -50,5 +48,3 @@ def run(self, params={}):
                 self.logger.error(f"Response was: {response.text}")
             raise PluginException(assistance="Please verify the connection details and input data.",
                                   cause=f"Error processing the Apex request: {rex}")
-
-        return {Output.SUCCESS : False}
```

Comment:
Do we want to remove this?

Reply:
This part of code was unreachable anyway, that's why we remove it :)